src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -11792,7 +11792,7 @@ public void LoadinganalyzerNetStandard13()
                 $@"warning AD0001: Analyzer 'TestAnalyzer' threw an exception of type 'System.NotImplementedException' with message '28'.
 System.NotImplementedException: 28
    at TestAnalyzer.get_SupportedDiagnostics()
-   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.AnalyzerExecutionContext.<>c__DisplayClass20_0.<ComputeDiagnosticDescriptors_NoLock>b__0(Object _)
+   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerManager.AnalyzerExecutionContext.<>c__DisplayClass21_0.<ComputeDiagnosticDescriptors_NoLock>b__0(Object _)
```

Comment:
test depends on exact callstack (including closures) that an exception is reported in.  So a change in the logic there ended up breaking this baseline in a very minor fashion.